client/app/telecom/telephony/billingAccount/orderAlias/nonGeographical/telecom-telephony-billing-account-orderAlias-nonGeographical.controller.js

```diff
@@ -109,7 +115,13 @@ angular.module('managerApp').controller('TelecomTelephonyAliasOrderNonGeographic
       ]);
     }
     const form = _.pick(this.form, filter);
-    form.offer = 'alias';
+
+    // check if SDA redirection only is displayed
+    if (this.showSDASelector) {
```

Comment:
```suggestion
    form.offer = this.showSDASelector ? this.offer : 'alias';
```

Reply:
It's not the same value (showSDASelector is a boolean).